atlite/convert.py

```diff
@@ -561,6 +566,10 @@ def pv(cutout, panel, orientation, clearsky_model=None, **params):
         'azimuth': 0.0} or a callback function with the same signature
         as the callbacks generated by the
         'atlite.pv.orientation.make_*' functions.
+    tracking : int 0, 1, 2:
+        0 for no tracking, default
+        1 for 1-axis vertical tracking
+        2 for 2-axis tracking
```

Comment:
I am wondering whether we should rather take the arguments 
* False, 
* 'v' or 'vertical' 
* 'h' or 'horizontal' 
* 'full'

where horizontal is not supported right now. This would make integration in the future easier. Could you change this?

Reply:
or instead of full "hv" or "vh"

Reply:
good idea indeed! I have not included 'horizontal' as that is not supported by the code, but should be now easier to integrate it in the future

Reply:
Sorry to jump in here without actually reviewing the code but this comment caught my attention. I think we should prioritize including 1-axis horizontal tracking. 
In short, horizontal 1-axis tracking is widely used in the market today (about 50% of new PV power plants use it) while 2-axis or vertical tracking is not used at all. 
I understand that this PR only includes vertical and 2-axes (and that is great, thanks for the work @enrilizaga ) but I think it would look weird in the next release if only exotic options are included by not the dominant "horizontal 1-axis tracking".

Reply:
Thanks @martavp. I understand your point, but I am not sure why that would prevent this PR from being added into the code? Horizontal 1-axis tracking may be more widely used, but it is also more complex to implement and outside my scope of work. PS: would be great if you could share the reference re PV panels market share by tracking type!

Reply:
@enrilizaga no worries, the PR is great (!) and this missing feature would not prevent it from being merged. But since you already have experience with the math, would you mind lifting the implementation to the full features stack? That would make it perfect. I had a try with horizontal tracking in #220 but due to missing time, I did not dig down any further. There you also find a reference (https://www.nrel.gov/docs/fy13osti/58891.pdf) for the hsat.

Reply:
Got it, thanks for the clarification. When is next release? I could look into it at some point in the next month or so but I am currently time constrained.

Re the market share, thanks for sharing that reference. Unfortunately ITRPV does not seem to have data on horizontal vs vertical 1-axis tracking separately and just reports all as "1-axis tracking"?

Reply:
No pressure on the release date. If you can make it this year, that would be great. Otherwise I could also spend some hours, likely not before January on this. To give you a proper hint on the implementation I'd have make some research, which is atm not possible for me...

Reply:
Hi @enrilizaga I'm pretty sure that ITRPV only mentions "1-axis tracking" assuming that is horizontal tracking. Vertical tracking is not used at all in commercial utility-scale applications. 

I'm also time constrained until January, but I can also find some time there to make suggestions. I have also talked to @Parisra and she might be able to put some time into implementing this.